app/cinema/bar.py

```diff
@@ -0,0 +1,4 @@
+class CinemaBar:
+    @staticmethod
+    def sell_product(product: str, customer: object) -> None:

```

Comment:
`object` is bad type annotation as everything in Python is object. Better specify that type

```suggestion
    def sell_product(product: str, customer: Customer) -> None:

```